fritz

```diff
@@ -157,17 +172,18 @@ def run(args):
                 str(Path(__file__).parent.absolute())
             )
         if args.traefik:
-            # fix host for traefik
+            # fix host for Traefik
             dc["services"]["web"]["labels"][2] = dc["services"]["web"]["labels"][2].replace(
                 "<host>",
                 fritz_config["skyportal"]["server"]["host"]
             )
+        else:
+            # not running behind Traefik? then publish port 5000 to 9000
+            dc["services"]["web"]["ports"] = ["9000:5000"]
```

Comment:
Should these port numbers be grabbed from config?

Reply:
Sure, fixed